cli/internal/util/task_output_mode.go

```diff
@@ -0,0 +1,34 @@
+package util
```

Comment:
I went with the `util` package for this file because the `status.go` file feels similar in nature. Does that sound right?

Reply:
I touched on this in the PR description, but, I needed to put this in another package to avoid the circular dependency in the `turbo_json.go` validation. 

From a structural perspective, it also feels better elsewhere since it's not necessarily about `runcache` output so much as it is about the display of task output. It's true all of that output currently lives in `runcache`, but, ideologically this feels distinct enough to live in another package.